src/software/backend/input/network/networking/proto_multicast_listener.cpp

```diff
@@ -17,8 +18,8 @@ SSLVisionClient::SSLVisionClient(boost::asio::io_service& io_service,
     catch (const boost::exception& ex)
     {
         LOG(WARNING) << "There was an issue binding the socket to the endpoint when"
-                        "trying to connect to the SSL Vision multicast address. This may"
-                        "be due to another instance of the the SSLVisionClient running"
+                        "trying to connect to the multicast address. This may be due"
+                        "to another instance of the the ProtoMulticastListener running"
```

Comment:
Repeated 'the' in comments. Might be worth just fixing now even though it was there previously.

Reply:
done